orc/src/main/java/org/apache/iceberg/orc/ORCSchemaUtil.java

```diff
@@ -70,20 +70,19 @@ public TypeDescription type() {
   private static final String ICEBERG_ID_ATTRIBUTE = "iceberg.id";
   private static final String ICEBERG_REQUIRED_ATTRIBUTE = "iceberg.required";
 
-  private static final String ICEBERG_BINARY_TYPE_ATTRIBUTE = "iceberg.binary-type";
-  private static final String ICEBERG_INTEGER_TYPE_ATTRIBUTE = "iceberg.integer-type";
-  private static final String ICEBERG_FIELD_LENGTH = "iceberg.length";
+  public static final String ICEBERG_BINARY_TYPE_ATTRIBUTE = "iceberg.binary-type";
```

Comment:
Can we add a brief description javadoc on the now public constants?

Reply:
Done